docs/tasks.md

```diff
@@ -0,0 +1,23 @@
+# Bolt Tasks
+
+This repo includes bolt tasks.  The available tasks are enumerated here.
+
+## gplt::create_control_repo_from_production_env
+
+This task creates a git repo using the contents from the given node's
+puppet production environment.  Therefor, the specified node should be
+the primary puppet master.
+
+The built in documentation for the task can be seen by running the following:
+```
+bolt task show gplt::create_control_repo_from_production_env
+```
+
+Typical usage on an ABS instance would use the following bolt pattern:
+```
+bolt task run \
+  --no-host-key-check \
+  --private-key ~/.ssh/id_rsa-acceptance \
```

Comment:
need --user root

Reply:
I believe that bolt defaulted to `--user root` if it was not specified.

Ref: https://puppet.com/docs/bolt/latest/bolt_configuration_options.html#ssh-transport-configuration-options

Reply:
When I ran it without --user root it tried to ssh into the box as randell...  which failed.  And frankly that doc doesn't sound right. Why would you want it to ssh as root instead of as the current user by default.   Either way they either have a bug in their code or documentation or the --private-key changes the default user.  Or could be a new feature to a very recent release... I am probably not on the latest.  I would add it in to the documentation to be safe, and to guard from future bolt changes.